docs/generate.js

```
@@ -0,0 +1,331 @@
+    let y;
+    if (continued == null) { continued = false; }
+    switch (this.type) {
+      case 'example':
+        this.setStyle(doc);
+        
+        // translate all points in the example code to 
+        // the current point in the document
+        doc.moveDown();
+        doc.save();
+        doc.translate(doc.x, doc.y);
+        var { x } = doc;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)